libpromises/processes_select.c

```diff
@@ -1268,7 +1266,8 @@ static FILE *OpenUcbPsPipe(void)
         if (!cmd)
         {
             Log(LOG_LEVEL_WARNING, "Could not execute \"%s\", extra process "
-                "information not available.", ps_cmd);
+                "information not available. Process command line length may be "
```

Comment:
Can you break the literal string on the punctuation? That way grepability of messages is not affected by folding long strings.